Makefile

```diff
@@ -71,7 +71,8 @@ test:
 	nosetests -x -v --nocapture --nologcapture --with-coverage --cover-html --cover-package=execution_engine2 test/ee2_SDKMethodRunner_test.py
 	nosetests -x -v --nocapture --nologcapture --with-coverage --cover-html --cover-package=execution_engine2 test/ee2_MongoUtil_test.py
 	nosetests -x -v --nocapture --nologcapture --with-coverage --cover-html --cover-package=execution_engine2 test/ee2_server_test.py
-	nosetests -x -v --nocapture --nologcapture --with-coverage --cover-html --cover-package=execution_engine2 test/ee2_load_test.py
+	#nosetests -x -v --nocapture --nologcapture --with-coverage --cover-html --cover-package=execution_engine2 test/ee2_load_test.py
```

Comment:
we should let travis run load test. it's the only integration test we have for some endpoints. if kafka code breaks that test, it's worth to figure it out.

Reply:
I added it back in, see build https://travis-ci.org/kbase/execution_engine2/builds/627475530